lib/vmdb/deprecation.rb

```diff
@@ -1,7 +1,7 @@
 module Vmdb
   class Deprecation
     def self.instance
-      @instance ||= ActiveSupport::Deprecation.new("D-release", "ManageIQ").tap { |d| d.behavior = default_behavior }
+      @instance ||= ActiveSupport::Deprecation.new("F-release", "ManageIQ").tap { |d| d.behavior = default_behavior }
```

Comment:
So, what you're saying is that we need to change the darga branch to say "Euwe release"